api.planx.uk/modules/webhooks/service/updateTemplatedFlowEdits/index.ts

```diff
@@ -0,0 +1,14 @@
+export const updateTemplatedFlowEdits = async (
+  flowId: string,
+  templatedFrom: string,
+  data: any,
+) => {
+  // TODO
+  // 1. Fetch flows.data where id = templatedFrom
+  // 2. JSON diff against event `data`
+  // 3. Filter diff for nodes tagged 'customisation'
+  // 4. Transform diff to `templated_flow_edits` data shape
+  // 5. Update `templated_flow_edits`
```

Comment:
Intentional TODOs ! For now, this function simply returns the arguments passed to it.